tests/export/html/test_field_code.py

```diff
@@ -7,11 +7,58 @@
 )
 
 
-from pydocx.openxml.packaging import MainDocumentPart
+from pydocx.openxml.packaging import MainDocumentPart, StyleDefinitionsPart
 from pydocx.test import DocumentGeneratorTestCase
 from pydocx.test.utils import WordprocessingDocumentFactory
 
 
+class HeadingTestCase(DocumentGeneratorTestCase):
```

Comment:
Maybe I should rename the other class to drop the FieldCode, since the module is test_field_code?